File: src/test/java/TestMybatis.java
import com.lq.bean.HistoryRecord;
import com.lq.bean.Message;
import com.lq.bean.Result;
import com.lq.service.AddService;
import com.lq.service.QueryService;
import com.lq.util.ResultUtil;
import org.junit.Test;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

import java.util.List;

/**
 * Created by LiQian_Nice on 2018/3/8
 */
public class TestMybatis {

    private static Logger logger= LoggerFactory.getLogger(TestMybatis.class);
    @Test
    public void one(){
        String command="看";
        QueryService queryService=new QueryService();
        List<Message> messageList=queryService.queryMessageList(command);
        for (Message message:messageList){
            logger.info(message.getCommand());
        }
    }

    @Test
    public void two(){
        HistoryRecord historyRecord=new HistoryRecord();
        historyRecord.setCommand("测试");
        historyRecord.setCreatetime("测试");
        AddService addService=new AddService();
        addService.addRecord(historyRecord);
    }

    @Test
    public void three(){
        QueryService queryService=new QueryService();
        List<HistoryRecord> historyRecordList=queryService.queryHistoryRecordList("查");
        for (HistoryRecord historyRecord:historyRecordList){
           logger.info(historyRecord.getCommand()+historyRecord.getCreatetime());
        }
    }

    @Test
    public void four(){

        QueryService queryService=new QueryService();
        boolean flag=queryService.querySameHistroyRecord("22423");
        logger.info(flag+"");

    }

    }

File: src/test/java/TestMybatis.kt
import com.lq.bean.HistoryRecord
import com.lq.service.AddService
import com.lq.service.QueryService
import org.junit.Test
import org.slf4j.LoggerFactory

/**
 * Created by LiQian_Nice on 2018/3/10
 *
 */
class TestMybatis{

    private val logger = LoggerFactory.getLogger(TestMybatis::class.java)

    @Test
    fun one() {
        val command = "看"
        val queryService = QueryService()
        val messageList = queryService.queryMessageList(command)
        for (message in messageList) {
            logger.info(message.command)
        }
    }

    @Test
    fun two() {
        val historyRecord = HistoryRecord()
        historyRecord.command = "测试"
        historyRecord.createtime = "测试"
        val addService = AddService()
        addService.addRecord(historyRecord)
    }

    @Test
    fun three() {
        val queryService = QueryService()
        val historyRecordList = queryService.queryHistoryRecordList("查")
        for (historyRecord in historyRecordList) {
            logger.info(historyRecord.command + historyRecord.createtime)
        }
    }

    @Test
    fun four() {

        val queryService = QueryService()
        val flag = queryService.querySameHistroyRecord("22423")
        logger.info(flag.toString() + "")

    }

}